### PR TITLE
[ossm] Update to OSSM 2.6

### PR DESCRIPTION
**Pull request by mkralik3**
Ref: https://issues.redhat.com/browse/OSSM-6453
* updated to OSSM 2.6
* removed 4.13 job (unsupported for OSSM 2.6)
* updated Gateway API CRDs
* changed firewatch assignee from rcernich to mkralik
* added waiting for Kiali is ready

### CODE REVIEW

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build pull-ci-maistra-maistra-test-tool-main-lint pull-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-images pull-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.17-lp-interop-images

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.17-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.17-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.17-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse pull-ci-maistra-maistra-test-tool-main-build pull-ci-maistra-maistra-test-tool-main-lint pull-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-images pull-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-images pull-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.17-lp-interop-images periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-ibmcloud

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud

**Comment by mkralik3** [comment]
Removed 4.13 job

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

File: ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -85,7 +84,8 @@ tests:
         ]
       REGION: us-west-2
       ROSA: "true"
-      SMCP_VERSION: "2.5"
+      SMCP_OPERATOR_VERSION: 2.6.0
```
Comment (FilipB) [inline ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml:87]: Is this needed?

Comment (mkralik3) [reply ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml:87]: yes, if we merge some test specific for a newer operator, e.g. 2.6.1, it needs to be skipped in the lpinterop, e.g. : https://github.com/maistra/maistra-test-tool/blob/main/pkg/tests/non-dependant/olm_webhooks_test.go#L20

**Review by FilipB — APPROVED** [review]

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-ibmcloud periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-rosa-classic-servicemesh-rosa-aws

**Pull request merged**